src/Model/Scope/Condition.php

```diff
@@ -201,8 +201,24 @@ public function toQueryArguments(): array
                         }
 
                         // support for conditions on aggregates
-                        if (preg_match('~^(\w+)\(([\w *]*)$~', $field, $matches)) {
-                            $field = $value ? $refModel->action('fx', array_slice($matches, 1)) : $refModel->action('exists');
+                        if (is_string($field) && str_contains($field, '(')) {
```

Comment:
fix force push!

Reply:
So we either converto count here or we use the expression conversion you commented below

Reply:
definitely here, we want even want to remove this completely very soon :)

Reply:
Now this adjusted